### DISCUSSION TITLE
Actual submission of tax documents

### DISCUSSION THREAD

**Discussion by topherbuckley** [eprbell/rp2#92] [General]
First off, THANK YOU! I was so happy to find this repo and what looks like a pretty active community. Also curious why US and JP are the current countries, but VERY glad they are, as these are precisely what I need :). Guessing someone else is a JP expat and was equally frustrated with the existing paid tools for this.  

I didn't find this in the README or FAQ, but please correct me if I'm missing it. What is the intended way of making use of the output ODS files for the actual submission of tax docs for the US? Are you copy/pasting everything into a PDF version of the 8949 or just writing in a line to refer to a printed version of the ODS file? I know Koinly does the PDF generation, but the amount of printed pages is unwieldy. Hoping to hear successful cases of using a single line to refer to the printed ODS file as this is what I do for other stocks/etc.

**Comment by eprbell** [comment]
Thanks! Really happy to hear about positive experiences with RP2. The reason for US and JP is random: I wrote the US plugin and @macanudo527 wrote the JP one. There is some momentum developing for Canada, so maybe that will be next :)

As for your question on forms, fair disclaimer: none of this is tax or financial advice. RP2 doesn't generate the final forms and it's recommended to check with a tax professional. I can tell you that my tax advisor copies every single line into the final forms (and then I keep the RP2 outputs as a record in case of audit). However note that not everything goes into form 8849: see [this FAQ](https://github.com/eprbell/rp2/blob/main/docs/user_faq.md#which-crypto-tax-forms-to-file) for more on this. Hope this helps!

**Reply by topherbuckley** [reply]
Alrighty, thanks for the info. I don't think my case warrants consultation with a tax professional, so I'll try my luck with submitting a printed version of the ODS file (or some parts of it). If the IRS really wants it copy/pasted on to a 8949, I'll work out some automation for moving this info to a pdf and add that here if others are interested in using it (with the same caveats you gave about).
